src/main/java/org/mitre/synthea/world/agents/Person.java

```diff
@@ -168,7 +168,7 @@ public Person(long seed) {
   }
 
   /**
-   * Retuns a random double.
+   * Returns a random double.
```

Comment:
Rather than having the random number related methods as public Person methods, I think it would be cleaner if they were all gathered together into an interface (e.g. RandomProvider) that Person implements. Methods that need access to randomness but not other aspects of Person could then just accept a RandomProvider instead of Person and this would make it clear which methods need Person access vs just Randomness.

Reply:
An interface like RandomProvider is a good idea. I'll work on that.

Reply:
I called the interface `RandomNumberGenerator` because I feel like `Provider` is overloaded with meaning, e.g. healthcare provider.